pages/research_software/software_releases.md

```diff
@@ -0,0 +1,66 @@
+In a software development life cycle this phase is located between testing & quality assurance and maintenance (see diagram below).
+
+![Release phase in software development life cycle](../../assets/img/release-in-sw-dev-life-cycle.png)
+
+Releasing a software means to create a snapshot of the current state of of your software and assigning a version to that snapshot.
+It is recommended to use [Semantic Versioning](https://semver.org/) as the versioning schema for the version strings.
+Semantic Versioning makes sure that users of your software understand the impact of your released software.
+Depending on whether it is a major, minor or patch release users can assess how severe the changes are when moving from one version to another.
+
+## Authors and Copyright
+
+Within this snapshot the authors and copyright holders of the files included need to be given.
+Giving credit to all contributors involved is also a good habit.
+It is a good practice to have a copyright statement in each of the files included.
+Tools like [REUSE](https://reuse.software/) can be used to check this.
+
+## Licenses
+
+A license manages the rights of the copyright holders and the users of the software.
+If no license is given all rights are reserved by the copyright holders and no rights are granted to others.
+It is recommended to release your software as open source software.
+In brief, for open source software rights like usage, modifications and distribution are granted to the users of your software.
+This involves choosing an appropriate open source software license from the list of [OSI approved licenses](https://opensource.org/licenses) beforehand.
+Each file in the release needs to have a license statement which can be verified by tools like [REUSE](https://reuse.software/).
+On top, the repository should include a file like `License.md` that contains the license text of the license used.
+
+## Citations
+
+You might also want to provide a file in the [Citation File Format (CFF)](https://citation-file-format.github.io/) like `CITATION.cff` to give citation information about your software and make sure that other can cite your software releases properly.
+
+## Changelog
+
+Semantic Versioning gives you a rough idea about the impact the release might have and the severity of changes when moving from one version to another.
+Nonetheless, it is inevitable to provide a standarised [Changelog](https://keepachangelog.com/en/1.1.0/) to users that explains in detail which changes where applied to a particular release.
+You can generated Changelogs from your commit messages with Changelog Generators like [git-cliff](https://www.conventionalcommits.org/en/v1.0.0/).
+Commit messages need to follow specific conventions.
+Writing good commit messages is essential for users to know what has been changed and why.
+It is recommended to follow common advice about writing [good commit messages](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/).
```

Comment:
We should probably mention other versioning schemes, such as calendar-based.
Committing is not about doing software releases, right?